src/_routing/_components/RootStackNavigator.tsx

```diff
@@ -15,17 +15,18 @@ export type TRootParams = Record<TRootRoutes, undefined>;
 const RootStack = createNativeStackNavigator<TRootParams>();
 
 export const RootStackNavigator = () => {
-  const { data: isPolicyApproved = false, isLoading } = useGetAsyncStorageValue<boolean>(StorageKey.IsPolicyApproved);
```

Comment:
😢

Reply:
I know, maar basically is dit wel wat ge wou he? :-p 
Nu zit ge niet meer met het `async` gegeven.

Maar RIP uw hook ⚰️